mobile/bindings.go

```diff
@@ -75,7 +75,7 @@ func Start(extraArgs string, rpcReady Callback) {
 
 	// Load the configuration, and parse the extra arguments as command
 	// line options. This function will also set up logging properly.
-	loadedConfig, err := lnd.LoadConfig(shutdownInterceptor)
+	loadedConfig, err := lnd.LoadConfig(*shutdownInterceptor)
```

Comment:
Why do we want to dereference here? Can't we just pass a pointer in?

Reply:
I initially tried passing it directly to figure out the LoadConfig does not accpet and reference to Interceptor. I was not sure which all aspects of LND would be using the loadConfig as it seems very important to parse the user configs, hence i decided to go the safe way and passed a de-referenced interceptor

https://github.com/lightningnetwork/lnd/blob/9e8b9ccd4c72f33e162f9b84b0f4d1f68e855984/config.go#L569

Reply:
You can bump `LoadConfig` to accept a pointer 👍